src/modules/dashboard/components/Whitelist/WhitelistAddresses/WhitelistMembersListExtraContent.tsx

```diff
@@ -46,10 +62,15 @@ const WhitelistMembersListExtraContent = () => {
           ],
         }}
       >
-        {/* @TODO:  Add interaction to remove an address from the list */}
-        <Button appearance={{ theme: 'ghost', size: 'small' }}>
+        <ActionButton
+          appearance={{ theme: 'ghost', size: 'small' }}
+          submit={ActionTypes.UPDATE_WHITELIST}
+          error={ActionTypes.UPDATE_WHITELIST_ERROR}
+          success={ActionTypes.UPDATE_WHITELIST_SUCCESS}
```

Comment:
This should be named the other way around: `WHITELIST_UPDATE`

That way we can more easily read/recognize what this action is about. Basically the pattern is `<COMPONENT-NAME>_<ACTION-NAME>_<OPTIONAL-STATUS>`